### PR TITLE
i18n(es): translate `overriding-components`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Translated `overriding-components` to spanish.

### CODE REVIEW

**Review by K1ri86 — APPROVED** [review]
Nice job @dreyfus92 😃

File: docs/src/content/docs/es/guides/overriding-components.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+title: Sobreescribiendo Componentes
+description: Aprende como sobreescribir los componentes integrados de Starlight para agregar elementos personalizados a la interfaz de tu sitio de documentación.
+sidebar:
+  badge: New
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/overriding-components.md:5]:
```suggestion
  badge: Nuevo
```

File: docs/src/content/docs/es/guides/overriding-components.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    <footer>Construido con Starlight 🌟</footer>
+  ) : (
+    <Default {...Astro.props}>
+      <slot />
+    </Default>
+  )
+}
+```
+
+Aprende más sobre el renderizado condicional en la [Guía de Sintaxis de Astro](https://docs.astro.build/es/core-concepts/astro-syntax/#dynamic-html).
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/overriding-components.md:133]:
```suggestion
Aprende más sobre el renderizado condicional en la [Guía de Sintaxis de Astro](https://docs.astro.build/es/core-concepts/astro-syntax/#html-dinamico).
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por sus revisiones @K1ri86 y @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Comment by kevinzunigacuellar** [comment]
Hay algunos fixes en los links #833. 

> Note: Hold merge until #833 is merged.

**Comment by dreyfus92** [comment]
> Hay algunos fixes en los links #833.
> 
> > Note: Hold merge until #833 is merged.

Ahorita me pongo a aplicar los cambios.

File: docs/src/content/docs/es/guides/overriding-components.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+</style>
+```
+
+Aprende más sobre todas las props disponibles en la [Referencia de Sobreescripciones](/es/reference/overrides/#props-de-componentes).
+
+### Solo sobreescribe en páginas específicas
+
+Los cambios de componentes se aplican a todas las páginas. Sin embargo, puedes renderizar condicionalmente usando los valores de `Astro.props` para determinar cuando mostrar tu UI personalizada, cuando mostrar la UI por defecto de Starlight, o incluso cuando mostrar algo completamente diferente.
+
+En el siguiente ejemplo, un componente que sobreescribe el [`Footer`](/es/reference/overrides/#footer-1) de Starlight muestra "Construido con Starlight 🌟" solo en la página de inicio, y de otra manera muestra el footer por defecto en todas las otras páginas:
```
Comment (delucis) [inline docs/src/content/docs/es/guides/overriding-components.md:111]: One more link fix if someone wants to PR it!

```suggestion
En el siguiente ejemplo, un componente que sobreescribe el [`Footer`](/es/reference/overrides/#footer) de Starlight muestra "Construido con Starlight 🌟" solo en la página de inicio, y de otra manera muestra el footer por defecto en todas las otras páginas:
```

Comment (kevinzunigacuellar) [reply docs/src/content/docs/es/guides/overriding-components.md:111]: I can send a PR!

**Pull request merged**